### PR TITLE
correct order of model paths

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: env-hooks/99.ed_object_models.bash [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
     local model_paths_string=""
     for dir in $unique_model_paths
     do
         if [[ "$dir" == "$model_path" ]]
         then
+            # You want to have the main model path at the end
             continue
         fi
-        model_paths_string="$dir:$model_paths_string"
+        model_paths_string="$model_paths_string:$dir"
```
Comment (ar13pit) [inline env-hooks/99.ed_object_models.bash]: Do it this way:
`model_paths_string=${model_paths_string:+${model_paths_string}:$dir}`

File: env-hooks/99.ed_object_models.bash [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
     done
-    if [ -n $model_paths_string ]
+
+    model_paths_string="$model_paths_string:$model_path"
+    model_paths_string=${model_paths_string:1} # remove leading colon, which is always there
```
Comment (ar13pit) [inline env-hooks/99.ed_object_models.bash]: Here as well. You won't have to remove the leading colon then.

File: env-hooks/99.ed_object_models.bash (hunk 1 above)
Comment (MatthijsBurgh) [reply env-hooks/99.ed_object_models.bash]: I don't know what it is supposed to do, but it gives me empty strings.

Comment (ar13pit) [reply env-hooks/99.ed_object_models.bash]: Its a type of Parameter expansion. Check this:
https://unix.stackexchange.com/a/102899

It shouldn't give you empty strings. I have tested this formulation. `$dir` should get appended to the list. In the installation of CUDA, this formulation is used to add bin to PATH and set LD_LIBRARY

**Pull request merged**